hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```diff
@@ -6762,11 +6768,6 @@ objects:
       matchLabels:
         api.openshift.com/managed: 'true'
     resourceApplyMode: Sync
-    resources:
-    - apiVersion: v1
-      kind: Namespace
-      metadata:
-        name: openshift-sre-sshd
```

Comment:
Removing it from this section will delete the namespace

Reply:
I moved the ns file to a different location.

Reply:
That doesn't matter. As the SSS is in "Sync" mode, it will still delete it, even if it's re-added to another SSS. Each SSS is separate in that way.